packages/bun-vscode/assets/bun.lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "JSON schema for bun.lock files.",
+  "allowTrailingCommas": true,
+  "type": "object"
+}
```

Comment:
in future we can type this better like the actual intention of the schema, but for now just using it for `allowTrailingCommas`

Reply:
Maybe you could even submit this upstream in https://github.com/microsoft/vscode/tree/main/extensions/json-language-features. You can still add a more detailed JSON schema later here or in the [JSON Schema Store](https://www.schemastore.org/json/). They’ll both be applied.